nucliadb_texts/src/writer.rs

```diff
@@ -220,7 +204,9 @@ impl TextWriterService {
     pub fn open(config: &TextConfig) -> NodeResult<Self> {
         let field_schema = TextSchema::new();
         let index = Index::open_in_dir(&config.path)?;
-        let writer = index.writer_with_num_threads(1, TANTIVY_INDEX_ARENA_MEMORY)?;
+        let writer = index
+            .writer_with_num_threads(1, TANTIVY_INDEX_ARENA_MEMORY)
```

Comment:
Is this the correct env var? TANTIVY_INDEX_ARENA_MEMORY is `6_000_000`. Do we have a different one to define threads and not memory?

Reply:
oops, I'm dumb -- ignore.